### PR TITLE
Add files via upload2

### PR DESCRIPTION
**Pull request by xdan-arch**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
:white_check_mark: A review job has been created and sent to the PullRequest network.  
*****  
Check the status or cancel PullRequest code review [here](https://daniel-b0af.pullrequest.tech/dash/1/xDanielA/test-repo/reviews).

**Review by ghost — COMMENTED** [review]
blahasdfsadfsdfasadsfdsfasdfsafds
[](#2380303c-4448-440e-a82e-ec07a781d5c5 "PullRequest Meta [Do Not Modify]")
*****
Was this helpful? [Yes](https://daniel-b0af.pullrequest.tech/reviews/ratings/2380303c-4448-440e-a82e-ec07a781d5c5/true) | [No](https://daniel-b0af.pullrequest.tech/reviews/ratings/2380303c-4448-440e-a82e-ec07a781d5c5/false)

File: test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+console.log("A");
+debugger;
```
[the remark this answers is not in the document]
Comment (xdan-arch) [reply test.js:2]: a real reply

**Comment by xdan-arch** [comment]
this looks legit!

**Pull request merged**